lib/assets/data_table/main.js

```diff
@@ -116,7 +123,14 @@ export function init(ctx, data) {
       orderBy(newKey) {
         this.loading = true;
         const [key, order] = reorder(this.data.content.order_by, this.data.content.order, newKey)
-        ctx.pushEvent("order_by", { key, order });
+        const label = key ? this.data.content.columns[key].label : null
+        ctx.pushEvent("order_by", { key, order, label});
```

Comment:
It looks like we are sending the label to the server, just to get it back. Any reason we cannot compute the label based on `columns` and `order_by`?

Reply:
We can and we should! The reason is my habit of doing liveview things server-side only. Done :)